spec/rubocop/cop/rspec/contain_exactly_spec.rb

```diff
@@ -45,17 +45,14 @@
     RUBY
   end
 
-  it 'flags `contain_exactly` with no arguments' do
+  it 'does not autocorrect `contain_exactly` with no arguments' do
     expect_offense(<<-RUBY)
       it { is_expected.to contain_exactly }
-                          ^^^^^^^^^^^^^^^ Prefer `be_empty` when matching an empty collection.
+                          ^^^^^^^^^^^^^^^ Prefer `match_array` when matching array values.
```

Comment:
I was wondering if modifying as per this offense would seem to be an offense of `RSpec/MatchArray` and might confuse people, especially those who are just starting to use RSpec?

Reply:
Me too. Would we just not register any offense then?

Reply:
Well, perhaps it would be better to leave that violation to the 'RSpec/BeEmpty' rather than to those cops, or to change the offense message.

```
Prefer `eq` or `be_empty` when matching empty collection.
```

Reply:
I chose the first, “leave that offense to the 'RSpec/BeEmpty' cop”